github.com.conf.template

```diff
@@ -25,25 +25,53 @@ server {
     proxy_request_buffering off;
     chunked_transfer_encoding on;
 
+    # We need a resolver configured so we can dynamically look up domains when
+    # redirecting. We make it an env var so that we can switch it out in prod
+    # for the Digital Ocean resolver
+    resolver ${RESOLVER};
 
     # only opensafely, opensafely-core, and opensafely-actions orgs can be accessed
     # location regex cannot match on query parameters
+
+    # basic git operations
     location ~ ^/(${ORGS})/[^/]+/info/refs {
         if ($args !~ "service=git-(upload|receive)-pack") { return 404; }
         proxy_pass https://github.com;
         proxy_redirect default;
     }
 
+    # basic git operations
     location ~ ^/(${ORGS})/[^/]+/git-(upload|receive)-pack {
         proxy_pass https://github.com;
         proxy_redirect default;
     }
 
+    # allow ssh keys to be retreived for a user
     location ~ ^/[^/]+\.keys {
+        limit_except GET { deny all; }
         proxy_pass https://github.com;
         proxy_redirect default;
     }
 
+    # allow release artifcats to be download from specific repo only
+    location ~ ^/opensafely-core/backend-server/releases/download {
+        limit_except GET { deny all; }
+        proxy_pass https://github.com;
+        proxy_redirect default;
+        # releases download redirects to an S3 bucket, which is not accessible from
```

Comment:
```suggestion
        # `releases/download` redirects to an S3 bucket, which is not accessible from
```
I /think/ this makes the sentence slightly easier to parse, but wdyt?

Reply:
Sure, I think I orginally intended 'release downloads', but this works too